main.py

```diff
@@ -423,6 +428,18 @@ def write_file(self) -> None:
             file.seek(0)
             file.write(f"{self.color_mode}\n{self.units}")
 
+    def check_for_updates(self) -> None:
+        # Check github for newer version
+
+        self.api_response = requests_get("https://api.github.com/repos/Futura-Py/weather/releases/latest")
+        self.latest_tag = self.api_response.json()["tag_name"].removeprefix("v")
+
+        if VERSION != self.latest_tag:
+            self.doupdate = askyesno("Update available!", "Do you want to update to the newest version?", master=self)
```

Comment:
Make it clear it needs to be done manually and that theyre just being redirected.

Reply:
Maybe something like "Note: You will be redirected to (url)" where the URL is the release changelog page? Assets for the app would be right below the changelog, and we can include a link to it.
This can work, right? I hope this does it!

Reply:
done